BUILDING.md

```diff
@@ -33,25 +33,36 @@ You can follow these instructions to build an nGraph Python wheel containing bot
 Prepare System:
 
     # apt update
-    # apt install python3 python3-pip python3-dev
-    # apt install build-essential cmake curl clang-3.9 git zlib1g zlib1g-dev libtinfo-dev
+    # apt install python3 python3-pip python3-dev python-virtualenv
+    # apt install build-essential cmake curl clang-3.9 git zlib1g zlib1g-dev libtinfo-dev unzip autoconf automake libtool
 
-Clone nGraph's `v0.10.0` tag, build and install it into `$HOME/ngraph_dist`:
+Clone nGraph's `v0.13.0` tag, build and install it into `$HOME/ngraph_dist`:
 
-    $ git clone -b 'v0.10.0' --single-branch --depth 1 https://github.com/NervanaSystems/ngraph.git
+    $ cd # Change directory to where you would like to clone nGraph sources
+    $ git clone -b 'v0.13.0' --single-branch --depth 1 https://github.com/NervanaSystems/ngraph.git
     $ mkdir ngraph/build
     $ cd ngraph/build
-    $ cmake ../ -DNGRAPH_USE_PREBUILT_LLVM=TRUE -DCMAKE_INSTALL_PREFIX=$HOME/ngraph_dist -DNGRAPH_ONNX_IMPORT_ENABLE=TRUE
+    $ cmake ../ -DCMAKE_INSTALL_PREFIX=$HOME/ngraph_dist -DNGRAPH_ONNX_IMPORT_ENABLE=TRUE -DNGRAPH_USE_PREBUILT_LLVM=TRUE 
     $ make
     $ make install
 
+Prepare a Python virtual environment for nGraph (recommended):
+
+    $ mkdir -p ~/.virtualenvs && cd ~/.virtualenvs
+    $ virtualenv -p $(which python3) nGraph
+    $ source nGraph/bin/activate
+    (nGraph) $ 
+
 Build Python package (Binary wheel) for nGraph:
 
-    $ cd ngraph/python
-    $ git clone --recursive -b allow-nonconstructible-holders https://github.com/jagerman/pybind11.git
-    $ export PYBIND_HEADERS_PATH=$PWD/pybind11
-    $ export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist
-    $ python3 setup.py bdist_wheel
+    (nGraph) $ cd # Change directory to where you would like to clone nGraph sources
```

Comment:
Probably you meant: "Change directory to where you *have cloned* nGraph sources"

Reply:
Copy-paste error :)